pubspec.yaml

```
@@ -7,18 +7,31 @@ environment:
+  formz: ^0.6.1
+  go_router: ^13.2.0
+  hive: ^2.2.3
+  hive_flutter: ^1.1.0
   intl: ^0.19.0
+  neopop: ^1.0.2
+  path_provider: ^2.1.2
 
 dev_dependencies:
+  bloc_test: ^9.1.7
+  build_runner: ^2.4.8
   bloc_test: ^10.0.0
```

YAMLlint (1.35.1)

[error] duplication of key "bloc_test" in mapping

(key-duplicates)